packages/zowe-explorer-ftp-extension/CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ All notable changes to the "zowe-explorer-ftp-extension" extension will be docum
 
 ### Bug fixes
 
+- Updated dependecies for technical currency. [#3786](https://github.com/zowe/zowe-explorer-vscode/issues/3786)
```

Comment:
```suggestion
- Updated dependencies for technical currency. [#3786](https://github.com/zowe/zowe-explorer-vscode/issues/3786)
```

Reply:
Resolved in 172cfe573ba7cff7df22d3b3c4cdf4e9fc773e81